docs/commerce-api.yaml

```
@@ -256,15 +258,109 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/Components/Schemas/ErrorResponse'
+                $ref: '#/components/schemas/ErrorResponse'
               examples:
                 serverError:
                   value:
                     error_code: SERVER_ERROR
                     error_message: "서버 내부 오류가 발생했습니다. 잠시 후 다시 시도해주세요."
                     error_data:
-Components:
-  Schemas:
+
+  /v1/orders/{order_id}/payment:
```

```suggestion
  /v1/orders/{order_id}:
```
로 해서 결제 부분 필드랑 주문상태만 업데이트 해주는게 더 레스트풀해보여요

---

피드백 감사합니다 반영하였습니다.
[959d564](https://github.com/HoBaeBang/commerce/pull/6/commits/959d564cbc64d6b516bc3d40b4a6f5b4182fbacd)